pkg/controller/cyclenoderequest/transitioner/transitions.go

```diff
@@ -106,6 +109,23 @@ func (t *CycleNodeRequestTransitioner) transitionPending() (reconcile.Result, er
 
 	// get instances inside cloud provider node groups
 	nodeGroupInstances := nodeGroups.Instances()
+
+	forceCheckAsgConfigAnnotations := t.cycleNodeRequest.Annotations[ForceAsgConfigCheckAnnotation]
+	if forceCheckAsgConfigAnnotations == "true" {
```

Comment:
Use `strconv.ParseBool()` which will parse a lot of different possible true/false inputs

Reply:
Goog point, even though it's cycle that adds this annotation and its value is controlled, and we don't expect users to add those annotations manually with values like `True`. Changes committed.

Reply:
imo still handle the error, e.g. if there's an error just assume it's false.

Reply:
hmm actually that would be the case, ignore my previous statement